dts/arm/nxp/nxp_lpc55S6x_common.dtsi

```diff
@@ -210,6 +211,15 @@
 		status = "okay";
 		label = "RNG";
 	};
+
+	wwdt0: watchdog@c000 {
+		compatible = "nxp,lpc-wwdt";
+		reg = <0xc000 0x1000>;
+		interrupts = <0 0>;
+		status = "disabled";
+		clk-divider = <1>;
```

Comment:
Is not the clock divider 4 ? If yes, it should be read int the macro above using DT_PROP()

Reply:
This clock divider is different from the fixed prescaler in the macro above.